### ISSUE TITLE
3.1.0-a1 & 3.1.0-a2 compile error

### ISSUE THREAD

**Issue by mia0x75** [phalcon/cphalcon#12706]
```
cphalcon-3.1.0-alpha2/build/php7/64bits/phalcon.zep.c:24810:13: error: static declaration of ‘zephir_init_static_properties_Phalcon_Tag’ follows non-static declaration
 static void zephir_init_static_properties_Phalcon_Tag(TSRMLS_D) {
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from cphalcon-3.1.0-alpha2/build/php7/64bits/phalcon.zep.c:175:0:
cphalcon-3.1.0-alpha2/build/php7/64bits/phalcon.zep.h:18084:6: note: previous declaration of ‘zephir_init_static_properties_Phalcon_Tag’ was here
 void zephir_init_static_properties_Phalcon_Tag(TSRMLS_D);
      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Makefile:194: recipe for target 'phalcon.lo' failed
```

**Comment by sergeyklay** [comment]
@mia0x75 Yes, I've seen this. Right now (before release) may be fixed manually:

**`ext/phalcon/tag.zep.h`**
```diff
- void zephir_init_static_properties_Phalcon_Tag(TSRMLS_D);
+ static void zephir_init_static_properties_Phalcon_Tag(TSRMLS_D);
```

**Comment by sergeyklay** [comment]
Fixed in the `3.1.x` branch.
